.circleci/config.yml

```
@@ -15,7 +13,7 @@ jobs:
       - run: git submodule sync
       - run: git submodule update --init
       - run: pip install --user tox
-      - run: tox -e py36,codecov
+      - run: tox -e py36,proxito,codecov
```

```suggestion
      - run: tox -e py36,proxito,search,codecov
```

This should call the tests for `search` as well.